ingestion-data/production/dataset-config/tornadoes-2024-dow-refl-greenfield.json

```
@@ -0,0 +1,68 @@
+{
+    "collection": "tornadoes-2024-dow-refl-greenfield",
+    "title": "DOW7 Reflectivity (Greenfield, IA)",
+    "description": "Horizontal Reflectivity values (dBZ) collected by DOW7 from the Greenfield, Iowa EF-4 tornado on May 21, 2024 at 20:41:08.",
```

```suggestion
    "description": "Horizontal reflectivity values (dBZ) from the EF-4 Greenfield, IA tornado on May 21, 2024 collected by DOW7. Scan taken at 20:41:08.",
```

I don't know how important this is but this is what veda-config has